packages/node_modules/nav-frontend-skjema-style/src/input-panel.less

```diff
@@ -71,6 +71,11 @@
     }
   }
 
+  &--error {
```

Comment:
Sørg for å bruk variabler fra nav-frontend-core når fargene angis, slik som her:
https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-skjema-style/src/validering.less

Reply:
Har lagt inn css variabler. Ellers burde det være lagt inn på todo listen å gå igjennom alle less filene og erstatte fargekoder med less variabler. Filen `input-panel.less` har konsekvent bare rgb fargekoder.